pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -1238,6 +1238,114 @@ spec:
 	}
 }
 
+// TestReconcileWithResolver checks that a PipelineRun with a populated Resolver
```

Comment:
docstring here I think needs to be updated

Reply:
👍